docs/toc.js

```diff
@@ -1,26 +1,23 @@
 module.exports = {
   toc: [
-    {
-      title: '📕 Why Storybook?',
-      pathSegment: 'why-storybook',
-      type: 'link',
-    },
+    
     {
       title: '🚀 Get started',
       pathSegment: 'get-started',
       type: 'menu',
       children: [
-        {
-          pathSegment: 'introduction',
-          title: 'Introduction',
-          type: 'bullet-link',
-        },
         {
           pathSegment: 'install',
           title: 'Install',
           type: 'bullet-link',
           description: 'Install the Storybook package in your project',
         },
+        {
```

Comment:
@kylegach the placement of "Why Storybook?" at the very top of the hierarchy was intentional. Can we change the position back?

Why SB is an ancillary document for folks that want a technical (read: _text_ document) version of why to use Storybook instead of browsing our home page.

Reply:
We discussed this offline and settled on keeping _Why Storybook?_ in the _Get started_ menu of the TOC, but moving it to be first, so that there's a better narrative flow from that page to the install page.